src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/topology/TestKafkaConsumer.java

```diff
@@ -55,6 +58,15 @@ public TestKafkaConsumer(final String topic, final Properties properties) {
         checkDestination = false;
     }
 
+    public TestKafkaConsumer(final String topic, final Properties properties, long kafkaMessagePollTimeout) {
+        this(topic, null, properties);
+        checkDestination = false;
+        this.kafkaMessagePollTimeout = kafkaMessagePollTimeout;
+    }
+
+    /**
+     * Starts kafka consumer.
```

Comment:
Kafka is the surname of the Czech novelist and usually starts with the capital letter https://en.wikipedia.org/wiki/Apache_Kafka

Reply:
done